tests/realtikvtest/importintotest/import_into_test.go

```diff
@@ -1239,3 +1239,26 @@ func (s *mockGCSSuite) TestBadCases() {
 	err := s.tk.QueryToErr(sql)
 	require.ErrorContains(s.T(), err, "panic occurred during import, please check log")
 }
+
+func (s *mockGCSSuite) TestImportIntoWithFK() {
+	content := []byte(`1,1
+	2,2`)
+	s.server.CreateObject(fakestorage.Object{
+		ObjectAttrs: fakestorage.ObjectAttrs{
+			BucketName: "foreign-key-test",
+			Name:       "child.csv",
+		},
+		Content: content,
+	})
+	tempDir := s.T().TempDir()
+	filePath := path.Join(tempDir, "file.csv")
+	s.NoError(os.WriteFile(filePath, content, 0644))
```

Comment:
why write to local, seems not used

Reply:
Nice catch. It's not used and has been removed.